src/DSharp.Script/src/System/Collections/CollectionHelpers.js

```diff
@@ -46,4 +68,47 @@ function values(obj) {
 
 function keyCount(obj) {
     return keys(obj).length;
-}
\ No newline at end of file
+}
+
+var contains = createFallbackFunction("contains", function (obj, value) {
+    return obj.indexOf(value) >= 0;
+});
+
+var insert = createFallbackFunction("insert", function (obj, index, value) {
+    obj.splice(index, 0, value);
+});
+
+var clear = createFallbackFunction("clear", function (obj) {
+    obj.length = 0;
+});
+
+function addRange(obj, range) {
+    if (typeof obj.addRange === "function") {
```

Comment:
We can update this to use the `createFallbackFunction` mechanism if you have time :)